### ISSUE TITLE
Whats missing exactly ? Any plans I guess ?

### ISSUE THREAD

**Issue by JohnyMac** [MissyMelissa/prophunt#1]
Soo, I forked this and I am willing to continue it.. What is remaining exactly ? Cause, well.. I can't figure out and I don't have enough players/friends to test it out xD
I can see some TODO's which I guess will start with those... But I need a better idea about the mode and what are you willing to do with it exactly.

**Comment by MissyMelissa** [comment]
Yeah, so I've added a bunch of TODO's before pushing this to github, I hope it should be clear!

Here's the gamemode rules on how it should work (as a quick summary):

1. Players join random teams when they join (hunter/props)
2. When there are 2 players in the game, the game will start immediately
3. If players spawn while a round is playing, they will join a random team and be put into spectator mode
4. Round starts, all hiders and props are moved into dimension 1, all seekers are blinded.
5. 30 seconds later, all hiders are moved back into dimension 0, all seekers are unblinded.
6. When a seeker kills a hider, the hider becomes a spectator. If all hiders are dead, seekers win.
7. 2 minutes later, if there are still hiders left, hiders win.
8. When a team wins, 15 seconds later, respawn any dead/spectating players, then restarts back from point 4.

The "kills a hider" part is a little bit vague, but this involves doing damage on the player's prop as well, not just the player's ped.

Speaking of which, the player's ped is currently visible as it seems that when you hide it (set transparency to 0), the game likes to crash at random points for some reason.

**Comment by JohnyMac** [comment]
Shouldn't there be like a map border or probably custom mapped arenas ?
Ik it's probably early to talk about this but just saying.

**Comment by MissyMelissa** [comment]
Yeah, you're right. I had that in the back of my mind too but didn't write it down 😊 

It should probably also be able to spawn players into a certain area with a certain size with certain props (eg. a regular old "map")

**Comment by JohnyMac** [comment]
Great, also, out of curiosity, why do you have the csproj and nuget packages file removed ? It's a hassle to setup new project, download packages and then add files.. could have been easier :D
